.claude/skills/beopsuny/scripts/fetch_law.py

```
@@ -1203,6 +1206,667 @@ def fetch_case_by_number(case_number: str):
     return fetch_case_by_id(case_id)
 
 
+# ============================================================
+# 체크리스트 기능
+# ============================================================
+
+def _generate_law_link(law_name: str, articles: list = None) -> str:
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)